corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java

```diff
@@ -0,0 +1,107 @@
+package ch.cordalo.corda.common.states;
+
+import net.corda.core.contracts.UniqueIdentifier;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+public class CordaloLinearStateTest {
+
+    private class UnderTest extends CordaloLinearState {
+
+        public UnderTest(UniqueIdentifier linearId) {
+            super(linearId);
+        }
+
+        public UnderTest(String externalId) {
+            super(externalId);
+        }
+
+        public UnderTest() {
+        }
+
+        @NotNull
+        @Override
```

Comment:
Test case missing for getParticiosnts() method. 

Add a party here and validate if part of participants

Reply:
done